internal/gen/controller.swagger.json

```diff
@@ -3699,8 +3699,7 @@
           "description": "The ID of the Auth Method that is associated with this Account."
         },
         "attributes": {
-          "type": "object",
-          "description": "The attributes that are applicable for the specific Account type."
```

Comment:
This comment disappearing implies that we misplaced a comment in a proto file somewhere, will investigate.

Reply:
Fixed